info-for-speakers.md

```diff
@@ -1,11 +1,8 @@
 ---
-title: Lead a session
+title: Information for speakers
```

Comment:
This is an internal navigation in a page. If you look at https://www.spaconference.org/spa2018/info-for-speakers.html, this is supports the menu down the left hand side, all under 'Lead a session'. 

If you want to change the way that works, that's fine but please explain why in the commit message.